server/common/api/models.py

```diff
@@ -12,6 +13,13 @@ class Class(models.Model):
 	minGuestCount = models.IntegerField(default = 4)
 	maxGuestCount = models.IntegerField(default = 8)
 	price = models.FloatField()
+	classRating = models.FloatField(default = 0.0,
+									validators = [
```

Comment:
Why are these lines indented so much? Should be the same level as "default = 0.0"